app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 //= require topbar
 //= require two-pane
 //= require vendor
-//= require about
+//= require statuses
```

Comment:
Does this mean we were loading the above javascript for each individual page? Can we do better and only load this for about and proxy pages now?